.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Warn and Close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 5 * * *'
+
+jobs:
+  warn_and_close_stails:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think it is good practice to use the GITHUB_TOKEN because if gets destroyed automatically after use in the workflow. We also need to audit making sure that we should use the GITHUB_TOKEN whenever possible.

Reply:
@Gauravtalreja1 Its not a new secret, it the toknen of `Github Action` user which is freely available in any Github repository. 

Since `Stale` is a a githubs own GHA I permitted GITHUB_TOKEN (meaning Github Action user) to do that for us. Just showing a love towards Github Action :)

Reply:
@jyejare ah, `secrets.GITHUB_TOKEN` is little misleading here, it feels like some secret added, so would suggest using `${{ github.token }}` here instead, which is equivalent i guess

Reply:
Not blocking. Overall both are same, Github community uses both of these options widely. So I wont block the PR for it and AFAIK I am good with whatever we have here.